test/test_framework.py

```diff
@@ -1663,31 +1729,43 @@ def test_pdb_properly_called(self, fake_stderr: io.StringIO):
         assert mock.call_count == 1
         assert mock.call_args == ((this_frame,), {})
 
-    def test_welcome_message(self, fake_stderr: io.StringIO):
+    def test_welcome_message(
+        self,
+        request: pytest.FixtureRequest,
+        monkeypatch: pytest.MonkeyPatch
```

Comment:
Per comment above, let's use a trailing comma in places like this to avoid multiline diffs in future:
```suggestion
        monkeypatch: pytest.MonkeyPatch,
```